www/js/survey/enketo-survey.js

```diff
@@ -100,7 +100,7 @@ angular.module('emission.survey.enketo.launch', [
 
   function initConfirmSurvey(opts) {
     return initSurvey({
-      form_location: 'json/20200420-01-trip-end-survey.json',
+      form_location: 'json/trip-end-survey.json',
```

Comment:
Is this what you want? I thought you wanted the most recent version of the survey...

Reply:
That's the most recent version. I removed all older forms.